otel-collector/Dockerfile

```diff
@@ -1,7 +1,7 @@
-ARG GOLANG_VERSION=1.22
+ARG GOLANG_VERSION
```

Comment:
env file with GOLANG_VERSION is missing

Reply:
I forgot to delete it in the previous PR, so it is still present in the repo